optimum/neuron/modeling_base.py

```diff
@@ -410,8 +407,11 @@ def _neuron_config_init(cls, config: "PretrainedConfig") -> "NeuronConfig":
         # Neuron config constructuor
         task = getattr(config, "task") or TasksManager.infer_task_from_model(cls.auto_model_class)
         task = TasksManager.map_from_synonym(task)
+        model_type = neuron_configs.get("model_type", None)
+        if not (model_type and model_type != "None"):
```

Comment:
No very easy to understand imo

Reply:
Tweaked a bit to avoid the conditional statement.